### PR TITLE
add js-polyfills@0.1.22 w/ npm autoupdate

### LINKED ISSUE #7508: [Request] Add inexorabletash-polyfill
**Library name:** js-polyfills
**Git repository url:** https://github.com/inexorabletash/polyfill
**npm package url(optional):** https://www.npmjs.com/package/js-polyfills
**License(s):** https://github.com/inexorabletash/polyfill/blob/master/LICENSE.md
**Official homepage:** https://github.com/inexorabletash/polyfill
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7508 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/inexorabletash/polyfill
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/js-polyfills
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 624
   - Count of watchers: 36
   - Count of forks: 127
 * NPM download stats (optional):
   - Downloads in the last day: 12
   - Downloads in the last week: 595
   - Downloads in the last month: 2489

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7508, cc @inexorabletash

### CODE REVIEW

File: ajax/libs/js-polyfills/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "Collection of Web polyfills.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/inexorabletash/polyfill.git"
+  },
+  "keywords": [
+    "polyfill",
+    "standards"
+  ],
+  "license": "Public Domain",
```
Comment (kennynaoh) [inline ajax/libs/js-polyfills/package.json]: @PeterDaveHello 
I don't see this in [license](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json).
Is this correct?

Comment (PeterDaveHello) [inline ajax/libs/js-polyfills/package.json]: Maybe you can confirm with author?

Comment (pvnr0082t) [inline ajax/libs/js-polyfills/package.json]: [tweetnacl](https://cdnjs.com/libraries/tweetnacl ) in cdnjs also use `Public Domain` licence. https://github.com/inexorabletash/polyfill/blob/master/LICENSE.md mention `Public Domain`.

Comment (pvnr0082t) [inline ajax/libs/js-polyfills/package.json]: Uh...What's wrong here? I didn't get it. @kennynaoh

**Comment by pvnr0082t** [comment]
@kennynaoh I have rebase to the latest master branch. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
So where did you solve the problem? Without any fixes but just rebase it, does this fix anything?

**Comment by pvnr0082t** [comment]
@PeterDaveHello So what should we solve? Confirm that which license the author use in [license list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)? Or just use `Public Domain`?

**Comment by PeterDaveHello** [comment]
@pvnr0082t you should ask @kennynaoh I just saw you both didn't end the conversation but make this PR good, doesn't this look strange?

**Comment by pvnr0082t** [comment]
ping @kennynaoh

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I have a question want to ask.
Whether the license must exist [here](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)?
Or it just explain clearly in [LICENSE](https://github.com/inexorabletash/polyfill/blob/master/LICENSE.md).
Thanks!

**Comment by PeterDaveHello** [comment]
@kennynaoh The problem is invalid, if the license is a string, it should be in the list, or it should be a object instead of a string to point to the license itsself.

**Comment by inexorabletash** [comment]
Sorry for my lack of attention here - I've been offline on vacation for a few weeks.

I've marked the license (in package.json) as "Unlicense" (which is in your list) and updated LICENSE.md; version is bumped to 0.1.22.

**Comment by pvnr0082t** [comment]
@kennynaoh I have updated licence field and library to v0.1.22. Please help me review again.

**Comment by kennynaoh** [comment]
@pvnr0082t 
Sorry for late review.
Please rebase to the latest master branch.
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh I have rebased it. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Pull request merged**